edx_proctoring/callbacks.py

```diff
@@ -152,12 +155,17 @@ def get(self, request, attempt_code):  # pylint: disable=unused-argument
             )
 
         update_exam_attempt(attempt['id'], last_poll_timestamp=timestamp, last_poll_ipaddr=ip_address)
+        time_remaining_seconds = get_time_remaining_for_attempt(attempt)
+        polling_interval = constants.DEFAULT_CLIENT_POLLING_INTERVAL
+        if time_remaining_seconds < constants.EXAM_CONCLUDING_INTERVAL:
```

Comment:
I think we need to switch to reduced polling when status is submitted in place of checking for time remaining.  Let's discuss if need be.

This may not affect anything, but what happens when time runs out in the exam?  Do they have to submit before end of exam, or is it auto-submitted when time runs out?